yarn.lock

```diff
@@ -537,10 +537,10 @@
     lodash "^4.17.11"
     path "^0.12.7"
 
```

Comment:
Did you mean to change this common.lib version?

Reply:
nope probably not, had to fix/suppress CVE issues and might have accidentally changed this. I can revert it

Reply:
please revert it. Thanks

Reply:
If you need any help understanding common-lib changes (if that helps) @paddybasi I was the one who did them so could give you relevant info should you need it